dist/linux/travis-ci.sh

```diff
@@ -1,12 +1,65 @@
-IFS='+' read -ra REFS <<< "$BLD"
+#! /bin/sh
+# This script is executed in the travis-ci environment.
 
-DBLD=${REFS[1]}
-PKG_DTAG=${REFS[0]}
-PKG_SHORTCOMMIT="$(echo $TRAVIS_COMMIT | cut -c1-10)"
+thismatrix=regular
+
+images=("ghdl/ghdl-tools:ubuntu-mcode"    \
+       "ghdl/ghdl-tools:ubuntu1404-llvm" \
+	   "ghdl/ghdl-tools:ubuntu1204-llvm" \
+       "ghdl/ghdl-tools:fedora-llvm-mcode")
+
+#--- image + identifier + compiler
+	   
+regular=("0+ubuntu+mcode" \
+         "3+fedora+llvm")
+
+nightly=("1+ubuntu1404+llvm-3.5" \
+         "2+ubuntu1204+llvm-3.8" \
+		 "3+fedora+mcode")
+
+#release=()
+
+#---
+
+task() {
+  printf "\033[33m[$1| BUILD] $2 \033[0m\n"
+
+  IFS='+' read -ra REFS <<< "$2"
+  DBLD=${REFS[2]}
+  wrkd="../wrk-$m"
+  cp -r ./ "$wrkd" && cd "$wrkd"
+  ./dist/linux/docker-buildtest.sh -i "${images[${REFS[0]}]}" -t "$1" -b "$DBLD" -f "ghdl-$PKG_VER-$DBLD-$PKG_TAG-${REFS[1]}-$PKG_SHORTCOMMIT.tgz"
+  cd "$cloned"
+  cp $(ls "$wrkd"/ghdl-*.tgz) ./
+  cp $(ls "$wrkd"/log.log) "./log_$m.log"
+}
```

Comment:
Certainly, I copypasted it from a previous version where I used a loop to modify `ghdl-*.tgz` names. But it is not needed now. I removed `ls`.